app/src/components/Meta.tsx

```diff
@@ -15,7 +15,7 @@ export const Meta = ({ title }: Props) => {
       .flat()
       .concat("Simplex")
       .map((part) => truncate(part, { length: 25, separator: " " }))
-      .filter((part) => part)
+      .filter(Boolean)
```

Comment:
Huh, `filter(Boolean)` is a neat trick, haven't seen that before. Kudos!

Reply:
Yeah I saw that in a gist somewhere randomly and thought it was brilliant way to filter out falsey values.